expandable-impl/src/token_tree.rs

```diff
@@ -759,12 +759,12 @@ mod tests {
                         Repetition(
                             Repetition {
                                 id: repetition 0,
-                                dollar: bytes(1..2),
+                                dollar: span($),
```

Comment:
TODO(scrabsha): carefully inspect this file and find why some spans are empty. This is probably a bug in the `GenericTokenTree` -> `TokenTree` convertion functions.